mirai-core-qqandroid/src/commonMain/kotlin/net/mamoe/mirai/qqandroid/contact/FriendImpl.kt

```diff
@@ -98,6 +99,24 @@ internal class FriendImpl(
         }
     }
 
+    @Suppress("DuplicatedCode")
+    override suspend fun nudge() {
+        require(bot.configuration.protocol == BotConfiguration.MiraiProtocol.ANDROID_PHONE) {
+            "Please use android phone protocol!"
```

Comment:
throw UnsupportedOperationException(' nudge is supported only with protocol ANDROID_PHONE')